# DP: Description: Correct linux paths for FHS
# DP: Author: Unknown
# DP: Upstream status: Debian-Specific
# DP: Status Details: GNU doesn't follow the FHS.
# DP: Date: Unknown

Index: b/sysdeps/unix/sysv/linux/paths.h
===================================================================
--- a/sysdeps/unix/sysv/linux/paths.h	2008-08-08 19:08:18.000000000 +0200
+++ b/sysdeps/unix/sysv/linux/paths.h	2008-08-08 19:09:03.000000000 +0200
@@ -67,7 +67,7 @@
 /* Provide trailing slash, since mostly used for building pathnames. */
 #define	_PATH_DEV	"/dev/"
 #define	_PATH_TMP	"/tmp/"
-#define	_PATH_VARDB	"/var/db/"
+#define	_PATH_VARDB	"/var/lib/misc/"
 #define	_PATH_VARRUN	"/var/run/"
 #define	_PATH_VARTMP	"/var/tmp/"
 
